bundles/io.openliberty.tools.eclipse.ui/src/io/openliberty/tools/eclipse/messages/Messages.properties

```diff
@@ -0,0 +1,90 @@
+###############################################################################
+# Copyright (c) 2023 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# NLS_ENCODING=UNICODE
+# NLS_MESSAGEFORMAT_VAR
+
+# DebugModeHandler
+multiple_server_env=More than one server.env file was found for the {0} project. Unable to determine which server.env file to use.
+
+# DevModeOperations
+start_no_project_found=An error was detected when the start request was processed. The object that represents the selected project was not found.
+start_already_issued=The start request was already issued on the {0} project. Use the stop action before you select the start action.
+start_general_error=An error was detected during the start request on the {0} project.
+
+start_container_no_project_found=An error was detected when the start in container request was processed. The object that represents the selected project was not found.
+start_container_already_issued=The start in container request was already issued on the {0} project. Use the stop action before you select the start action.
+start_container_general_error=An error was detected during the start in container request on the {0} project.
+
+stop_no_project_found=An error was detected when the stop request was processed. The object that represents the selected project was not found. When you use the Run Configuration launcher, be sure to select a project or project content first.
+stop_already_issued=Unable to process the stop request on the {0} project because either Liberty Tools did not previously issue a start request, or a stop request was already issued.
```

Comment:
```suggestion
stop_already_issued=Unable to process the stop request on the {0} project, either because Liberty Tools did not previously issue a start request or because a stop request was already issued.
```